src/pages/home/PackageList/Package/styles.ts

```diff
@@ -110,7 +110,7 @@ export const PackageListItemText = styled(ListItemText)({
   paddingRight: 0,
 });
 
-export const Description = styled(Typography)<{ theme?: Theme }>(({ theme }) => ({
+export const Description = styled(MuiText)<{ theme?: Theme }>(({ theme }) => ({
```

Comment:
I have to check this MuiText later on... for sure I have to omit the heading types.